### PR TITLE
feat(Image): add `onActive` to `toolbarRender`

### PR DESCRIPTION
**Pull request by madocto**
[中文版模板 / Chinese template](https://github.com/ant-design/ant-design/blob/master/.github/PULL_REQUEST_TEMPLATE/pr_cn.md?plain=1)

### 🤔 This is a ...

- [x] New feature
- [ ] Bug fix
- [ ] Site / documentation update
- [ ] Demo update
- [ ] Component style update
- [ ] TypeScript definition update
- [ ] Bundle size optimization
- [ ] Performance optimization
- [ ] Enhancement feature
- [ ] Internationalization
- [ ] Refactoring
- [ ] Code style optimization
- [ ] Test Case
- [ ] Branch merge
- [ ] Workflow
- [ ] Other (about what?)

### 🔗 Related issue link
close: #47948 

### 📝 Changelog

| Language   | Changelog |
| ---------- | --------- |
| 🇺🇸 English |  Image add `onActive` to `toolbarRender` for toggling images .      |
| 🇨🇳 Chinese |    Image 新增  `onActive` 到  `toolbarRender` 以切换图片 。   |

### ☑️ Self-Check before Merge

⚠️ Please check all items below before requesting a reviewing. ⚠️

- [x] Doc is updated/provided or not needed
- [x] Demo is updated/provided or not needed
- [x] TypeScript definition is updated/provided or not needed
- [x] Changelog is provided or not needed

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -120,7 +120,7 @@
     "rc-drawer": "~7.2.0",
     "rc-dropdown": "~4.2.0",
     "rc-field-form": "~2.4.0",
-    "rc-image": "~7.9.0",
+    "rc-image": "~7.10.0",
```
Comment (afc163) [inline package.json:123]:
```suggestion
    "rc-image": "~7.11.0",
```

**Comment by madocto** [comment]
本地单测没问题，找下什么原因

**Comment by madocto** [comment]
由于 rc-dialog@9.6.0 更新导致的快照变化，更新了快照

**Comment by madocto** [comment]
存在两个不同版本的 rc-dialog，rc-dialog@9.5.2 和 rc-dialog@9.6.0，要新开个 PR 把 rc-dialog@9.5.2 更新到 rc-dialog@9.6.0 吗？

File: package.json [hunk 2]
```diff
@@ -120,7 +120,7 @@
     "rc-drawer": "~7.2.0",
     "rc-dropdown": "~4.2.0",
     "rc-field-form": "~2.4.0",
-    "rc-image": "~7.9.0",
+    "rc-image": "~7.11.0",
```
Comment (afc163) [inline package.json:123]:
```suggestion
    "rc-image": "~7.10.0",
```

**Review by afc163 — DISMISSED** [review]

**Comment by afc163** [comment]
我先回到 5.10.0 了，只需要更新 snapshot 即可。

**Review by afc163 — APPROVED** [review]

**Pull request merged**